bin/anycablebility

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'docopt'
```

Comment:
I think that declarative approach is significant when it comes to building user interfaces. That's why I choose Docopt. It requires you to specify the actual help message that will be displayed to the end user to declare argument parsing rules.

> Also we can avoid the additional dependency.

Docopt is [relatively compact](https://github.com/docopt/docopt.rb/blob/master/lib/docopt.rb) (16KB) and it has well-established API.

> We can use OptionParse, which is a common practice.

Why is this important? Could you please expand on this?